checks/evaluation/branch_protection_test.go

```diff
@@ -18,563 +18,1691 @@ import (
+					Values: map[string]int{
+						"main2":                     1,
+						"numberOfRequiredReviewers": 1,
+					},
+				},
+				{
+					Probe:   "requiresCodeOwnersReview",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main2": 1,
+					},
+				},
+				{
+					Probe:   "requiresLastPushApproval",
+					Outcome: finding.OutcomeNegative,
+					Values: map[string]int{
+						"main2": 1,
+					},
+				},
+				{
+					Probe:   "requiresUpToDateBranches",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main2": 1,
+					},
+				},
+				{
+					Probe:   "runsStatusChecksBeforeMerging",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main2": 1,
+					},
+				},
+				{
+					Probe:   "requiresPRsToChangeCode",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main2": 1,
+					},
+				},
+			},
+			result: scut.TestReturn{
+				Score:        8,
+				NumberOfWarn: 3,
+				NumberOfInfo: 7,
+			},
+		},
+		{
+			name: "1 branch that is not protected",
+			findings: []finding.Finding{
+				{
+					Probe:   "blocksDeleteOnBranches",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "blocksForcePushOnBranches",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "branchesAreProtected",
+					Outcome: finding.OutcomeNegative,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "branchProtectionAppliesToAdmins",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "dismissesStaleReviews",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "requiresApproversForPullRequests",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main":                      1,
+						"numberOfRequiredReviewers": 1,
+					},
+				},
+				{
+					Probe:   "requiresCodeOwnersReview",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "requiresLastPushApproval",
+					Outcome: finding.OutcomeNegative,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "requiresUpToDateBranches",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "runsStatusChecksBeforeMerging",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+				{
+					Probe:   "requiresPRsToChangeCode",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"main": 1,
+					},
+				},
+			},
+			result: scut.TestReturn{
+				Score:        5,
+				NumberOfWarn: 1,
+			},
```

Comment:
similarly, if the branch is not protected, this would be a 0?

Reply:
Same as https://github.com/ossf/scorecard/pull/3759/files/00e20ba47fd1d1825b9204d898fc10c4c960cf56#r1478152843